### PR TITLE
Fix headers inclusion for BrainBit MacOS

### PR DESCRIPTION
**Pull request by mesca**

### CODE REVIEW

File: third_party/neurosdk/inc/win/celectrode-state-channel.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#ifndef CELECTRODE_STATE_CHANNEL_H
```
Comment (Andrey1994) [inline third_party/neurosdk/inc/win/celectrode-state-channel.h]: is it github bug that these files not shown as renamed or did you replace them by headers from macos?

**Comment by Andrey1994** [comment]
and is it the latest version of headers for macos?

File: third_party/neurosdk/inc/win/celectrode-state-channel.h (hunk 1 above)
Comment (mesca) [reply third_party/neurosdk/inc/win/celectrode-state-channel.h]:
```
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/c_ecg_channels.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/c_eeg_channels.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cbattery-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cbipolar-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cbridge-channels.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cchannel-helper.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cchannels.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cdevice.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/celectrode-state-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cemulation-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cfiltered-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/clistener.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cmems-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/corientation-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cparams.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cresistance-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/crespiration-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cscanner.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/csignal-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cspectrum-channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/cspectrum_power_channel.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/device_enumerator_wrapper.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/lib_export.h.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in third_party/neurosdk/inc/macos/sdk_error.h.
The file will have its original line endings in your working directory
```

**Comment by mesca** [comment]
> and is it the latest version of headers for macos?

Yes.

**Pull request merged**